src/controllers/redeploy.go

```diff
@@ -1,20 +1,45 @@
 package controllers
 
 import (
+	"errors"
+	"fmt"
+
 	"github.com/devclub-iitd/DeployBot/src/history"
+	"github.com/devclub-iitd/DeployBot/src/slack"
+	log "github.com/sirupsen/logrus"
 )
 
 func redeploy(callbackID string, data map[string]interface{}) {
+	log.Infof("service(%s) is being stopped", data["git_repo"].(string))
```

Comment:
I think you meant redeployed instead of stopped

Reply:
Stop hi tha, as in pehle stop hoti hai na

Reply:
wo toh stop routine me ho jaega log.